pages/news.html

```diff
@@ -88,6 +88,7 @@ <h1>NEWS</h1>
 <div class="container page-wrapper">
     <div class="container">
         <div class="row content-padding subpage-content">
+            <p><strong>20/12/2023</strong>Uyuni 2023.12 is now available! New products enabled! Check the <a href='pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p>
```

Comment:
```suggestion
            <p><strong>20/12/2023</strong>Uyuni 2023.12 is now available! Raspberry Pi OS 12 and Amazon Linux 2023 are now supported as clients, CLM filter by package build date and security fixes! Check the <a href='--/pages/stable-version.html#releasenotes'>release notes</a> for the full details!</p></p>
```

Reply:
Let's be a bit more explicit, and also you need to go one level up so the link works.